### PR TITLE
Investment updates

### PR DESCRIPTION
**Pull request by localastronaut**
Included project field and more details on how distribution works.

### CODE REVIEW

File: templates/artist/artist_application.html [hunk 1]
```diff
@@ -13,7 +13,13 @@ <h5>Want to learn more before signing up? Check out our <a href="{% url 'artist-
         <form action="" method="POST">
             {% csrf_token %}
             {{ form.as_p }}
+            <p>
+                I understand that PerDiem will exclusivly distribute the music project (free of charge) for the payback period listed in this form. All royalties (100%) will be split between Artist and Investors based on the artists terms.
```
Comment (RevolutionTech) [inline templates/artist/artist_application.html]: Typos:
```suggestion
                I understand that PerDiem will exclusively distribute the music project (free of charge) for the payback period listed in this form. All royalties (100%) will be split between the artist(s) and investors based on the artist(s)' terms.
```

File: templates/artist/artist_application.html [hunk 2]
```diff
@@ -13,7 +13,13 @@ <h5>Want to learn more before signing up? Check out our <a href="{% url 'artist-
...
+            </p>
             <input type="submit" value="Submit" />
         </form>
     </div>
+    <div>
+
+    </div>
```
Comment (RevolutionTech) [inline templates/artist/artist_application.html]: Why this block?

File: templates/email/welcome.email [hunk 3]
```diff
@@ -3,23 +3,23 @@
 {% block email_subject %}Welcome to PerDiem!{% endblock %}
 
 {% block plain_body %}
-    Thanks for joining PerDiem!
+    Thanks for your interest in supporting a new generation of artists and contributing to the progression fo their careers. We're excited to have you on board!
```
Comment (RevolutionTech) [inline templates/email/welcome.email]:
```suggestion
    Thanks for your interest in supporting a new generation of artists and contributing to the progression of their careers. We're excited to have you on board!
```

File: templates/email/welcome.email [hunk 4]
```diff
@@ -3,23 +3,23 @@
...
     {% endif %}
 
-    Discover artists on PerDiem:
+    You are now able to discover and invest in emerging artists through the platform. Please read over our FAQ and For Investors to get a better understanding of how investing works. We're constantly making updates and improving the site so be sure to subscribe and stay updated with all the latest info.
     {{ host }}{% url 'artists' %}
 {% endblock %}
 
 {% block html_body %}
-    <p>Thanks for joining PerDiem!</p>
+    <p>Thanks for your interest in supporting a new generation of artists and contributing to the progression fo their careers. We're excited to have you on board!</p>
```
Comment (RevolutionTech) [inline templates/email/welcome.email]: Same typo here

File: artist/forms.py [hunk 5]
```diff
@@ -25,6 +25,10 @@ class ArtistApplyForm(forms.Form):
             attrs={"placeholder": "We started playing music because..."}
         )
     )
+    project = forms.CharField(
+        label="Project Name",
+        widget=forms.TextInput(attrs={"placeholder": "Single/Album Name"}),
+    )
```
Comment (RevolutionTech) [inline artist/forms.py]: The addition of this requires an update to the `testArtistApplication()` test. Check out `artist/tests.py:ArtistWebTestCase.testArtistApplication()`.

File: templates/artist/artist_application.html (hunk 2 above)
Comment (localastronaut) [reply templates/artist/artist_application.html]: No reason. Accidentally left it there

File: artist/tests.py [hunk 6]
```diff
@@ -191,6 +191,7 @@ def testArtistApplication(self):
                     "We are a really cool heavy metal band. We mostly perform covers but are excited to "
                     "create an album, and we're hoping PerDiem can help us do that."
                 ),
+                "project": "$Album",
```
Comment (RevolutionTech) [inline artist/tests.py]: Was the $ sign a typo?

In following the patterns of what I'm seeing on the site, this seems like it should be the name of the album, which based on the line below you could say `Access Granted`:
```suggestion
                "project": "Access Granted"
```

**Pull request merged**